modules/unauthenticated-users-cluster-role-binding-con.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="unauthenticated-users-cluster-role-bindings-concept_{context}"]
+= Cluster role bindings for unauthenticated groups
+
+[NOTE]
+====
+Before {product-title} 4.16, unauthenticated groups were allowed access to some cluster roles. Clusters upgraded from versions before {product-title} 4.16 retain this access for unauthenticated groups.
```

Comment:
Per the [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update), update is the preferred term:
```suggestion
Before {product-title} 4.16, unauthenticated groups were allowed access to some cluster roles. Clusters updated from versions before {product-title} 4.16 retain this access for unauthenticated groups.
```